src/UserService/Program.cs

```diff
@@ -12,16 +12,26 @@ public static void Main(string[] args)
         {
             var configuration = new ConfigurationBuilder()
                 .AddJsonFile("appsettings.json")
-#if DEBUG
-                .AddJsonFile("appsettings.Development.json")
```

Comment:
Why did you deleted it? We have not any info in those files?

Reply:
Spartak said that  "Program.cs" needed only fol logger, so this changes won't affect on Production and Develop appsetings